test/xpu_api/numerics/c.math/cmath.pass.cpp

```diff
@@ -78,25 +78,25 @@ void test_abs()
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wabsolute-value"
 #endif
-    static_assert((std::is_same<decltype(dpl::abs((float)0)), float>::value), "");
-    static_assert((std::is_same<decltype(dpl::abs((int)0)), int>::value), "");
-    static_assert((std::is_same<decltype(dpl::abs((long)0)), long>::value), "");
-    static_assert((std::is_same<decltype(dpl::abs((long long)0)), long long>::value), "");
-    static_assert((std::is_same<decltype(dpl::abs((unsigned char)0)), int>::value), "");
-    static_assert((std::is_same<decltype(dpl::abs((unsigned short)0)), int>::value), "");
-    static_assert((std::is_same<decltype(dpl::abs((signed char)0)), int>::value), "");
-    static_assert((std::is_same<decltype(dpl::abs((short)0)), int>::value), "");
-    static_assert((std::is_same<decltype(dpl::abs((unsigned char)0)), int>::value), "");
-    static_assert((std::is_same<decltype(dpl::abs((char)0)), int>::value), "");
-    IF_DOUBLE_SUPPORT(static_assert((std::is_same<decltype(dpl::abs(Ambiguous())), Ambiguous>::value), ""))
-
-    static_assert(!has_abs<unsigned>::value, "");
```

Comment:
I guess `::value` is removed mistakenly (it is needed because has_abs is derived from `true_type/false_type`)

Reply:
You are right. Fixed.